_posts/2023-01-29-team43-visual-physical-reasoning.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+comments: true
+title: Visual Physical Reasoning
+author: Ian Galvez, James Youn
+date: 2023-01-29
+---
+
+> Our project will be based around the topic of **visual physical reasoning**. Essentially, visual physical reasoning deals with whether computers can answer "common sense" queries about an image, such as "how many chairs are at the table?", or "where is the red cube in relation to the purple ball?"
+
```

Comment:
Add `<!--more-->` here.
See this post: https://piazza.com/class/lch0yyp3khh5hn/post/165